wiki/Glossary/Local_song_offset/en.md

```diff
@@ -6,7 +6,7 @@ stub: true
 
 *For offset while beatmapping, see: [Offset](/wiki/Beatmapping/Offset)*
 
-**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the global offset to calculate a total offset.
+**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the global offset to calculate a total [offset](/wiki/Beatmapping/Offset).
```

Comment:
the link is incorrect, that page is not about a "total" offset.

```suggestion
**Local song offset** is an offset which moves the appearance of [hit objects](/wiki/Hit_object) in relation to the audio of individual [beatmaps](/wiki/Beatmap). This can be helpful for players who experience auditory or visual delays. Local song offset works alongside the [global offset](/wiki/Client/Options/Universal_offset) to calculate a total offset.
```